pymc/step_methods/hmc/base_hmc.py

```diff
@@ -205,10 +203,11 @@ def astep(self, q0):
                     point = DictToArrayBijection.rmap(info.state.q)
 
                 if self._num_divs_sample < 100 and info.state_div is not None:
-                    point = DictToArrayBijection.rmap(info.state_div.q)
+                    point_dest = DictToArrayBijection.rmap(info.state_div.q)
```

Comment:
@aseyboldt was this a typo? The `point_dest` was always `None`..

Reply:
I don't think it is always None. It should be non-None if the destination position contains nans or inf, unless I'm missing something...

Reply:
In line 196/194 you initialized `point_dest = None` but in the _before_ version of this code it was never assigned to anything.